pkg/autoscaling/estimator/percentile.go

```diff
@@ -137,9 +142,22 @@ func getCpuConfig(config map[string]string) *predictionconfig.Config {
 		marginFraction = "0.15"
 	}
 
+	initModeStr, exists := config["cpu-model-init-mode"]
+	initMode := predictionconfig.ModelInitModeLazyTraining
+	if !exists {
+		initMode = predictionconfig.ModelInitMode(initModeStr)
+	}
+
+	historyLength, exists := config["cpu-model-history-length"]
```

Comment:
How about moving the constant to a common file, like `EVPACaller-%s-%s` 、 `cpu-model-history-length`、`24h` etc.?

Reply:
> How about moving the constant to a common file, like `EVPACaller-%s-%s` 、 `cpu-model-history-length`、`24h` etc.?

Yes, we can do it later